learningeconomy/LearnCard
# [LC-198] Make clicking anywhere on a VC card flip it

[![workerB](https://img.shields.io/endpoint?url=https%3A%2F%2Fworkerb.linearb.io%2Fv2%2Fbadge%2Fprivate%2FU2FsdGVkX19tt5tGxugtioTYL7eD1FzNH57INPeuU%2Fcollaboration.svg%3FcacheSeconds%3D60)](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=lWNU5U-)
# Overview

#### 🎟 Relevant Jira Issues

[LC-198](https://welibrary.atlassian.net/browse/LC-198)

#### 📚 What is the context and goal of this PR?
Make clicking anywhere on a VC flip it

oh and also fixed some annoying circular dependency issues

#### 🥴 TL; RL:

#### 💡 Feature Breakdown (screenshots & videos encouraged!)

#### 🛠 Important tradeoffs made:

#### 🔍 Types of Changes


#### 💳 Does This Create Any New Technical Debt? ( If yes, please describe and [add JIRA TODOs](https://welibrary.atlassian.net/jira/software/projects/WE/boards/2) )

# Testing

#### 🔬 How Can Someone QA This?


- setup
  - link react-learn-card from `learncardapp/apps/learn-card-app/package.json` AND `learncardapp/packages/learn-card-base/package.json` 
  - `pnpm i` in learncardapp
  - `pnpm build` in `LearnCard/packages/react-learn-card` (might need to build additional packages or use npx/pnx build react-learn-card)
  - should be good to go, start learncardapp
- click on a credential to view it
- confirm that you can click anywhere to flip it
- confirm that if there are clickable items on the card, you can still click those without flipping the card
- repeat for certificate credential

#### 📱 🖥 Which devices would you like help testing on?


#### 🧪 Code Coverage


# Documentation

#### 📜 Gitbook


#### 📊 Storybook



# ✅ PR Checklist

### 🚀 Ready to squash-and-merge?:
